### ISSUE TITLE
Bluetooth turns off time to time

### ISSUE THREAD

**Issue by elementaryBot** [elementary/wingpanel-indicator-bluetooth#8]
I am using a bluetooth mouse. After some time (more or less 30 minutes) bluetooth turns off and I should turn on and pair the mouse again. 

Sometimes even the wing panel icon disappear and I have to go to settings, where it looks like in the attachement. and turn on off and on combination to get it work again.

It's started like one month ago more or less. (elementaryOS loki 0.4)

dmesg log after the turning off:
dmesg | grep -i blue
[    4.098872] thinkpad_acpi: rfkill switch tpacpi_bluetooth_sw: radio is unblocked
[    5.214065] Bluetooth: Core ver 2.21
[    5.214389] Bluetooth: HCI device and connection manager initialized
[    5.214784] Bluetooth: HCI socket layer initialized
[    5.214788] Bluetooth: L2CAP socket layer initialized
[    5.214794] Bluetooth: SCO socket layer initialized
[    6.529893] Bluetooth: BNEP (Ethernet Emulation) ver 1.3
[    6.529898] Bluetooth: BNEP filters: protocol multicast
[    6.529903] Bluetooth: BNEP socket layer initialized
[    7.346342] Bluetooth: hci0 command 0x0c03 tx timeout
[   15.342733] Bluetooth: hci0 sending initial HCI reset command failed (-110)
[   17.346193] Bluetooth: hci0 command 0x0c03 tx timeout
[   25.342193] Bluetooth: hci0 sending initial HCI reset command failed (-110)
[   97.754737] Bluetooth: hci0: read Intel version: 370710018002030d00
[   97.757726] Bluetooth: hci0: Intel Bluetooth firmware file: intel/ibt-hw-37.7.10-fw-1.80.2.3.d.bseq
[   97.965794] Bluetooth: hci0: Intel Bluetooth firmware patch completed and activated
[   98.023225] Bluetooth: RFCOMM TTY layer initialized
[   98.023231] Bluetooth: RFCOMM socket layer initialized
[   98.023240] Bluetooth: RFCOMM ver 1.11
[  101.354954] Bluetooth: HIDP (Human Interface Emulation) ver 1.2
[  101.354960] Bluetooth: HIDP socket layer initialized
[  101.355500] input: ThinkPad Bluetooth Laser Mouse as /devices/pci0000:00/0000:00:14.0/usb2/2-7/2-7:1.0/bluetooth/hci0/hci0:256/0005:17EF:6038.0004/input/input21
[  101.355767] hid-generic 0005:17EF:6038.0004: input,hidraw3: BLUETOOTH HID v2.00 Mouse [ThinkPad Bluetooth Laser Mouse] on 7c:7a:91:1a:4f:6d
[  495.374120] input: ThinkPad Bluetooth Laser Mouse as /devices/pci0000:00/0000:00:14.0/usb2/2-7/2-7:1.0/bluetooth/hci0/hci0:256/0005:17EF:6038.0005/input/input22
[  495.374402] hid-generic 0005:17EF:6038.0005: input,hidraw3: BLUETOOTH HID v2.00 Mouse [ThinkPad Bluetooth Laser Mouse] on 7c:7a:91:1a:4f:6d
[ 4879.599772] Bluetooth: hci0: read Intel version: 370710018002030d55
[ 4879.599777] Bluetooth: hci0: Intel device is already patched. patch num: 55
[ 4913.650428] input: ThinkPad Bluetooth Laser Mouse as /devices/pci0000:00/0000:00:14.0/usb2/2-7/2-7:1.0/bluetooth/hci0/hci0:256/0005:17EF:6038.0006/input/input23
[ 4913.651531] hid-generic 0005:17EF:6038.0006: input,hidraw3: BLUETOOTH HID v2.00 Mouse [ThinkPad Bluetooth Laser Mouse] on 7c:7a:91:1a:4f:6d

service bluetooth status: (after the bug)
● bluetooth.service - Bluetooth service
   Loaded: loaded (/lib/systemd/system/bluetooth.service; enabled; vendor preset: enabled)
   Active: active (running) since Út 2017-01-03 10:35:31 CET; 1h 21min ago
     Docs: man:bluetoothd(8)
 Main PID: 3254 (bluetoothd)
   Status: "Running"
   CGroup: /system.slice/bluetooth.service
           └─3254 /usr/lib/bluetooth/bluetoothd

led 03 11:55:23 Tomsgu bluetoothd[3254]: gatt-time-server: Input/output error (5)
led 03 11:55:23 Tomsgu bluetoothd[3254]: Not enough free handles to register service
led 03 11:55:23 Tomsgu bluetoothd[3254]: Not enough free handles to register service
led 03 11:55:23 Tomsgu bluetoothd[3254]: Sap driver initialization failed.
led 03 11:55:23 Tomsgu bluetoothd[3254]: sap-server: Operation not permitted (1)
led 03 11:55:23 Tomsgu bluetoothd[3254]: Endpoint registered: sender=:1.51 path=/MediaEndpoint/A2DPSource
led 03 11:55:23 Tomsgu bluetoothd[3254]: Endpoint registered: sender=:1.51 path=/MediaEndpoint/A2DPSink
led 03 11:55:47 Tomsgu bluetoothd[3254]: Can't get HIDP connection info
led 03 11:55:52 Tomsgu bluetoothd[3254]: connect error: Host is down (112)
led 03 11:55:56 Tomsgu bluetoothd[3254]: Can't get HIDP connection info

after wing panel crash:
led 03 12:00:17 Tomsgu bluetoothd[3254]: Can't get HIDP connection info
led 03 12:00:17 Tomsgu bluetoothd[3254]: Endpoint unregistered: sender=:1.51 path=/MediaEndpoint/A2DPSource
led 03 12:00:17 Tomsgu bluetoothd[3254]: Endpoint unregistered: sender=:1.51 path=/MediaEndpoint/A2DPSink
led 03 12:00:45 Tomsgu systemd[1]: Started Bluetooth service.

lspci:lspci
00:00.0 Host bridge: Intel Corporation Haswell-ULT DRAM Controller (rev 09)
00:02.0 VGA compatible controller: Intel Corporation Haswell-ULT Integrated Graphics Controller (rev 09)
00:03.0 Audio device: Intel Corporation Haswell-ULT HD Audio Controller (rev 09)
00:14.0 USB controller: Intel Corporation 8 Series USB xHCI HC (rev 04)
00:16.0 Communication controller: Intel Corporation 8 Series HECI #0 (rev 04)
00:19.0 Ethernet controller: Intel Corporation Ethernet Connection I218-V (rev 04)
00:1b.0 Audio device: Intel Corporation 8 Series HD Audio Controller (rev 04)
00:1c.0 PCI bridge: Intel Corporation 8 Series PCI Express Root Port 6 (rev e4)
00:1c.1 PCI bridge: Intel Corporation 8 Series PCI Express Root Port 3 (rev e4)
00:1d.0 USB controller: Intel Corporation 8 Series USB EHCI #1 (rev 04)
00:1f.0 ISA bridge: Intel Corporation 8 Series LPC Controller (rev 04)
00:1f.2 SATA controller: Intel Corporation 8 Series SATA Controller 1 [AHCI mode] (rev 04)
00:1f.3 SMBus: Intel Corporation 8 Series SMBus Controller (rev 04)
03:00.0 Network controller: Intel Corporation Wireless 7260 (rev 83)

Launchpad Details: [#LP1653661](https://bugs.launchpad.net/bugs/1653661) Tomáš Jakl - 2017-01-03 10:58:44 +0000

**Comment by danrabbit** [comment]
This seems like it's probably a problem with Bluez and not something we can fix in the indicator. Closing as "Out of scope"
